### ISSUE TITLE
Confirmation token is invalid

### ISSUE THREAD

**Issue by lucianosousa** [ricardovaleriano/call4paperz#98]
I just tried create an account and receive this error when I click on confirmation link
```
Resend confirmation instructions
1 error prohibited this user from being saved:

    Confirmation token is invalid
```

**Comment by vyper** [comment]
I receive same message!

**Comment by atilacamurca** [comment]
Me too.

And another thing, if I try to resend the validation token, the same token was sended to my e-mail.

I first try to login with Twitter, but no success. Then I tried the "old school" way. And the message above was shown. So I think that when my first attemp to login with Twitter fail, then other attemps failed too.

**Comment by raphaeljlps** [comment]
+1

**Comment by lucianosousa** [comment]
@ricardovaleriano can you merge the PR #101 to fix this?
Tks

**Comment by ricardovaleriano** [comment]
Should be solved by #101, can you guys check and repot any problems?
Thank you @lucianosousa!
